activesupport/lib/active_support/messages/serializer_with_fallback.rb

```diff
@@ -6,6 +6,10 @@ module ActiveSupport
   module Messages # :nodoc:
     module SerializerWithFallback # :nodoc:
       def self.[](format)
+        if format.match?(/message_pack/) && !defined?(ActiveSupport::MessagePack)
```

Comment:
Rubocop flags this with `Performance/StringInclude: Use String#include? instead of a regex match with literal-only pattern`, but `format` is a symbol, not a string.

I could work around using `format.to_s.include?("message_pack")`, but I'm more inclined to disable the `Performance/StringInclude` rule, since this is valid code.

Reply:
`format.start_with?("message_pack")` could do the trick.

Reply:
Or just `if format == :message_pack || format == :message_pack_allow_marshal`. A bit longer but very explicit.

Reply:
My thinking was to `require` when `format` included any mention of `message_pack` because tests may not catch a missing `require` (due to previous tests performing the `require`).  So if some `:message_pack_*` or `:*_message_pack` variant is added in the future, it would be covered.  (I admit that may be overkill though.)